src/elements/OcButton.vue

```diff
@@ -226,4 +244,16 @@ Every button has to have an accessible name. It cannot be provided by a text bet
     <oc-button variation="danger">Delete</oc-button>
   </div>
 ```
+
+### Coloring buttons with variation "raw"
```

Comment:
Could you add a `default` one as well here, so that all existing options are visible in one place?